### PR TITLE
Usability fixes

### PR DESCRIPTION
**Pull request by ingvard**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java [hunk 1]
```diff
@@ -168,14 +164,14 @@ private ReconciliationResult partitionReconciliationCheck(
...
         Set<String> cacheNames = null;
-        boolean fixMode = (boolean) FIX_MODE.defaultValue();
-        boolean verbose = (boolean) INCLUDE_SENSITIVE.defaultValue();
-        boolean console = (boolean) LOCAL_OUTPUT.defaultValue();
-        double loadFactor = (double) LOAD_FACTOR.defaultValue();
-        int batchSize = (int) BATCH_SIZE.defaultValue();
-        int recheckAttempts = (int) RECHECK_ATTEMPTS.defaultValue();
-        RepairAlgorithm repairAlg = (RepairAlgorithm) FIX_ALG.defaultValue();
-        int recheckDelay = (int) RECHECK_DELAY.defaultValue();
+        boolean fixMode = false;
```
Comment (sanpwc) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java]: Why not to rename it to repair?
and use REPAIR.defaultValue() here?

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java [hunk 2]
```diff
@@ -68,16 +67,17 @@
         String CACHES = "cacheName1,...,cacheNameN";
 
         String desc = "Verify whether there are inconsistent entries for the specified caches " +
-            "and print out the differences if any. Fix inconsistency if " + FIX_MODE + "argument is presented. " +
```
Comment (sanpwc) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java]: Don't forget to update control.sh_cache_help.output

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java [hunk 3]
```diff
@@ -194,23 +190,24 @@ private ReconciliationResult partitionReconciliationCheck(
...
-                        }
-                        catch (IllegalArgumentException e) {
-                            throw new IllegalArgumentException(
-                                "Invalid repair algorithm: " + strVal + ". The following " +
-                                "values can be used: " + Arrays.toString(RepairAlgorithm.values()) + '.');
+                        String peekedNextArg = argIter.peekNextArg();
+
+                        if (!PartitionReconciliationCommandArg.commands().contains(peekedNextArg)) {
+                            strVal = argIter.nextArg(
+                                "The repair algorithm should be specified. The following " +
```
Comment (sanpwc) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java]: Don't forget to update org.apache.ignite.internal.commandline.CommandHandlerParsingTest#testPartitionReconciliationArgumentsValidation

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java (hunk 1 above)
Comment (ingvard) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java]: REPAIR.defaultValue() -> returns default strategy. If this argument exists, fixMode = true.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java (hunk 2 above)
Comment (ingvard) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java (hunk 3 above)
Comment (ingvard) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java]: fixed

**Pull request closed without merging**